frontend/CLAUDE.md

```
@@ -61,6 +61,49 @@ mise exec:vue "[command]"
+/* PC */
+@media (min-width: 769px) {
+}
+```
+
+基本的にはスマホとpcだけで良いですが、タブレットでスマホレイアウトだと見づらい場合にはタブレット用のレイアウトを追加してください。
+
+# primeVueのコンポーネントの呼び出し
+
+基本的には`from 'primevue'`からまとめて呼び出す形にして、1行にまとめてください。
+
+### ok
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)